### DISCUSSION TITLE
Not able to use Quarkus OpenApi Generator Server 3.0.0-SNAPSHOT

### DISCUSSION THREAD

**Discussion by JayGhiya** [quarkiverse/quarkus-openapi-generator#698] [General]
Understanding from quarkus documentation that the libs that are in snapshot are being handled by quarkus cli automatically in terms of fetching from right registry/repository. My assumption might be wrong too. so tried using quarkus cli to update the dependency 
```
<dependency>
  <groupId>io.quarkiverse.openapi.generator</groupId>
  <artifactId>quarkus-openapi-generator-server</artifactId>
  <version>3.0.0-SNAPSHOT</version>
  </dependency>
```
It failed to fetch from snapshot repository/registry.
So tried using the last released version 2.4.1 but the properties seem to be unrecognised and it fails in generating server resources.

Here the application properties:

```
quarkus.openapi.generator.spec=archguard.json
quarkus.openapi.generator.base-package=org.acme.unoplat

```

**Comment by JayGhiya** [comment]
the project is open source - https://github.com/unoplat/unoplat-codebase-understanding/tree/main/code-with-quarkus

**Comment by ricardozanini** [comment]
@carlesarnal can you please take a look?

**Comment by maxandersen** [comment]
quarkus cli is not participating in how snapshots are handled.

That's up to your maven/gradle configuration.

**Reply by JayGhiya** [reply]
Understood so yea i understand usually you will add config did repos in.m2 for snapshots so if cli is not doing that could you guide me to right doc of quarkus to add right quarkus snapshot repo. My bad for assumption.

**Comment by ricardozanini** [comment]
@JayGhiya I don't think we publish snapshots yet. At least our CI for this plugin does not. Unless the Quarkus Platform does. I'm not familiar with this process thou. @gastaldi do you know?

**Reply by gastaldi** [reply]
You can add this workflow file to enable that: https://github.com/quarkiverse/quarkus-prettytime/blob/main/.github/workflows/snapshot-deploy.yml
